README.md

```diff
@@ -296,6 +296,7 @@ The player instance exposes the following methods, accessible when the player is
 | `mute()`               |         -          |     -     | Mute the volume                   |
 | `unMute()`             |         -          |     -     | Unmute the volume                 |
 | `seekTo(time)`         |      `Number`      |     -     | Seek to a current time in seconds |
+| `setSource(source)`    |      `String`      |     -     | Set the media source              |
```

Comment:
It should be added a mention to specify that it is only available for HTML5 video (and audio?)

You can get inspiration from similar examples in the README

https://github.com/vlitejs/vlite/blob/main/README.md?plain=1#L266-L270

Reply:
Should it be implemented also for other non-HTML5 providers?

Reply:
`videoId` parameter name is okay for Youtube, Vimeo and Dailymotion but not for HTML5 which have a video url. Needs to adapt